app/templates/index.html

```diff
@@ -46,6 +48,205 @@ <h4>LET’S CONNECT</h4>
                 </div>
             </div>
         </section>
+
+        <!--Middle section one-->
+        <section class="section-elements">
+            <div class="middle-section">
+                <div class="section-one-bg"></div>
+                <div class="section-block">
+                    <div class="section-title"><h2>Tonic</h2></div>
+                    <div class="section-frame">
+                        <div class="cannopy">CANOPY
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div>
+                        </div>
+                        <div class="backend">Back End Dev
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div> 
+                        </div>
+                        <div class="year">2015</div>
+                    </div>
+                    <div class="section-primary-text">
+                        <p>
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="section-tags">
+                        <button type="submit">html</button>
+                        <button type="submit">css</button>
+                        <button type="submit">javascript</button>
+                    </div>
```

Comment:
- [ ] You have made a great effort in listing the technologies used. However, according to the project requirements, you are required to list them using an un ordered/ ordered list and not divs and buttons.Kindly make this change.

**From the project requirements:**

![image](https://user-images.githubusercontent.com/86318284/185556488-850eb691-82e8-4fc1-aebd-aac4b82d53c7.png)


For a visual of what I mean, I will write you an example of how you are to do it:

```
<ul>
  <li>Html</li>
  ...your technologies here
</ul>
```

**Kindly make this change in also all the projects as I have realised the same trend there.**

Reply:
Sure, I will make the necessary changes
However, I got linter errors for combining li and images. 
Could you also provide assistance or a directive on how I should go about this?

Reply:
That's strange @kennankole as far as I know, it is not supposed to throw any error. Text me on slack we have a look at it together.